statics/styles.css

```diff
@@ -0,0 +1,3 @@
+/* #summary-window {
+    text-align: center;
+} */
```

Comment:
Do we need this?

Reply:
it is placeholder for future use. I think we are going to need to enhance UI a bit later.